draft-ietf-quic-manageability.md

```diff
@@ -898,6 +898,19 @@ are likely to fail, and are not recommended to use as a way to construe
 internals of QUIC's operation as those mechanisms can change, e.g., due to the
 use of extensions.
 
+## Handling ICMP Messages
+
+Datagram Packetization Layer PMTU Discovery (PLPMTUD) can be used by QUIC to
+probe for the supported PMTU. PLPMTUD optionally uses ICMP messages (e.g.,
+IPv6 Packet Too Big messages). Given know attacks with the use of ICMP messages,
+the use of PLPMTUD in QUIC has been designed to safely utilise received ICMP
+but do not rely upon them to function (see section 14.2.1. of
+{{QUIC-TRANSPORT}}).
+
+Networks are recommended to forward these ICMP messages and retain as much of
+the original packet as possible without exceeding the minimum MTU for the IP
+version when generating ICMP messages as recommended in {{(RFC1812, RFC4443)}}.
```

Comment:
Wow, I didn't realize that this syntax was even possible.  I might have to use that in future.

Reply:
Actually is not. The build is broken and I didn't check that yesterday. But would be nice...